src/components/Choropleth.vue

```diff
@@ -99,5 +109,11 @@ const finishUpdatingMap = () => {
     mapLoading.value = false;
 };
 
-watch([selectedFeatures, selectedIndicator], ([newFeatures]) => updateMap(newFeatures));
+watch(
+    [selectedFeatures, selectedIndicator, selectedAdminLevel],
+    ([newFeatures, , newAdminLevel], [, , oldAdminLevel]) => {
+        const doLockBounds = !!selectedCountryId.value || newAdminLevel !== oldAdminLevel;
```

Comment:
why do we need to change the bounds lock if the admin level has changed - would it remove the previous lock otherwise?

Reply:
yep i have gone with resetting the lock every update and then explicitly defining when we want to lock, think it is better if anything goes wrong that the bounds are the whole world